haskell/src/Lentille/GitLab/MergeRequests.hs

```diff
@@ -330,6 +331,18 @@ transformResponse host getIdentIdCB result =
             getMergedByIdent (ChangeOptionalMergedByMergedBy ident) = ident
             getMergedAt (ChangeOptionalMergedAtMergedAt ts) = ts
 
+        getChangeAbandonedEvent :: Change -> [ChangeEvent]
+        getChangeAbandonedEvent change =
+          [ (getBaseEvent change)
+              { changeEventId = "ChangeAbandonedEvent-" <> changeId change,
+                changeEventType = Just $ ChangeEventTypeChangeAbandoned ChangeAbandonedEvent,
+                -- GitLab Graph API does not report who closed a MR
+                changeEventAuthor = changeAuthor change,
+                changeEventCreatedAt = changeUpdatedAt change
+              }
+            | isClosed $ changeState change
```

Comment:
There are no guards L317 for ChangeCreatedEvent, is that expected?

Reply:
That's expected. This event is always generated.